src/test/java/plugins/GradlePluginTest.java

```diff
@@ -49,7 +49,8 @@ public void run_gradle_scirpt() {
 
         job.startBuild().shouldSucceed()
                 .shouldContainsConsoleOutput("Hello world!")
-                .shouldContainsConsoleOutput("gradle --quiet")
+                //reg expression gradle[\w\\/"]* --quiet
+                .shouldContainsConsoleOutput("gradle[\\w\\\\/\"]* --quiet")
```

Comment:
`gradle.*--quiet` seems to read better and be less fragile in case of other options will be placed before that.